torch/_dynamo/variables/builtin.py

```diff
@@ -1581,7 +1581,10 @@ def call_sum(self, tx: "InstructionTranslator", seq, start=_SENTINEL):
             if start is self._SENTINEL:
                 start = variables.ConstantVariable.create(0)
             items = seq.unpack_var_sequence(tx)
-            return BuiltinVariable(functools.reduce).call_function(
```

Comment:
We need to change some hardcoded handlers to use `SourcelessBuilder.create(tx, value)`.